lib/src/repository/repository.dart

```diff
@@ -3,6 +3,7 @@ import 'package:gql_exec/gql_exec.dart';
 import 'package:gql_link/gql_link.dart';
 
 class Repository {
+  // ignore: unused_element
```

Comment:
Mostly to get rid of a warning for unused elements but it can be removed if desired.

Reply:
This was something we talked about previously in a different PR where it was either we made this class abstract or use a private constructor to prevent it from being instantiable. I think I had a thumbs up on my comment so I went that way but it was giving my editor a blue squiggle warning. 

Preferences?

Reply:
No strong preference.  Just curious.  If there is no downside then it seems fine to me.

Reply:
This should be removed now. I had the way to declare this constructor not exactly the most correct way. Now it doesn't need the annotation to make the editor happy.